src/status_im/commands/handlers/loading.cljs

```diff
@@ -95,18 +99,36 @@
 
 (defn add-commands
   [db [id _ {:keys [commands responses autorun]}]]
-  (let [account    @(subscribe [:get-current-account])
-        commands'  (filter-forbidden-names account id commands)
-        responses' (filter-forbidden-names account id responses)]
-    (-> db
-        (assoc-in [id :commands] (mark-as :command commands'))
-        (assoc-in [id :responses] (mark-as :response responses'))
-        (assoc-in [id :commands-loaded] true)
-        (assoc-in [id :autorun] autorun))))
+  (let [account        @(subscribe [:get-current-account])
+        commands'      (filter-forbidden-names account id commands)
+        global-command (:global commands')
+        commands''     (apply dissoc commands' [:init :global])
+        responses'     (filter-forbidden-names account id responses)]
+    (cond-> db
+
+            (get-in db [:chats id])
+            (update-in [:chats id] assoc
+                       :commands (mark-as :command commands'')
+                       :responses (mark-as :response responses')
+                       :commands-loaded true
+                       :autorun autorun
+                       :global-command global-command)
```

Comment:
well we should have `@bot/command-name` when there is a namespacing issue on the command name within the current chat context, but we can't have global commands all over the place. It's simple just one that corresponds to `@bot` if they have it implemented, and the implementation is up to the developer, not us.